packages/catalog-search/src/FacetListBase.jsx

```diff
@@ -54,9 +59,14 @@ const FacetListBase = ({
       if (!items?.length) {
         return <span className="p-2 d-block">{NO_OPTIONS_FOUND}</span>;
       }
-
+      let isChecked = false;
       return items.map(item => {
-        const isChecked = isCheckedField ? item[isCheckedField] : !!item.value;
+        if (doRefinement) {
```

Comment:
`doRefinement` will always be `True` as per L119 and there will be no change in the existing flow except for the dropdowns where we want to not refine the data from Algolia.